kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Commands.java

```diff
@@ -0,0 +1,484 @@
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+        return System.getProperty("java.io.tmpdir");
+    }
+
+    public static String getLocalMavenRepoDir() {
+        for (String p : new String[] { "TESTS_MAVEN_REPO_LOCAL", "tests.maven.repo.local" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+
+        String mainBuildRepo = System.getProperty("maven.repo.local");
+        if (StringUtils.isNotBlank(mainBuildRepo)) {
+            return mainBuildRepo;
+        }
+
+        return System.getProperty("user.home") + File.separator + ".m2" + File.separator + "repository";
+    }
+
+    /**
+     * Get system properties starting with `quarkus.native` prefix, for example quarkus.native.builder-image
+     * 
+     * @return List of `-Dquarkus.native.xyz=foo` strings
+     */
+    public static List<String> getQuarkusNativeProperties() {
+        List<String> quarkusNativeProperties = System.getProperties().entrySet().stream()
+                .filter(e -> e.getKey().toString().contains("quarkus.native"))
+                .map(e -> "-D" + e.getKey() + "=" + e.getValue())
+                .collect(Collectors.toList());
+        return quarkusNativeProperties;
+    }
+
+    public static String getQuarkusPlatformVersion() {
+        for (String p : new String[] { "QUARKUS_PLATFORM_VERSION", "quarkus.platform.version" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+        logger.warn("Failed to detect quarkus.platform.version/QUARKUS_PLATFORM_VERSION, defaulting to getQuarkusVersion().");
+        return getQuarkusVersion();
+    }
+
+    public static String getQuarkusVersion() {
+        for (String p : new String[] { "QUARKUS_VERSION", "quarkus.version" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+        String failure = "Failed to determine quarkus.version. Check pom.xm, check env and sys vars QUARKUS_VERSION";
+        try (Scanner sc = new Scanner(new File(getBaseDir() + File.separator + "pom.xml"), StandardCharsets.UTF_8)) {
```

Comment:
Wouldn't this be more readable?
```suggestion
        try (Scanner sc = new Scanner(Path.of(getBaseDir(), "pom.xml"), StandardCharsets.UTF_8)) {
```

Reply:
Yes, didn't know that Scanner can use Path as well. Apparently it can since Java 10.